internal/gps/pkgtree/pkgtree.go

```diff
@@ -199,10 +199,17 @@ func fillPackage(p *build.Package) error {
 	var testImports []string
 	var imports []string
 	for _, file := range gofiles {
-		// Skip underscore-led files, in keeping with the rest of the toolchain.
-		if filepath.Base(file)[0] == '_' {
+		// Skip underscore-led or dot-led files, in keeping with the rest of the toolchain.
+		bPrefix := filepath.Base(file)[0]
+		if bPrefix == '_' || bPrefix == '.' {
 			continue
 		}
+
+		// Skip any directories that happened to get caught by glob
```

Comment:
It still is. My comment about "NOTE: this doesn't handle directories" no longer applies, as demonstrated by the attached test case.